content/post/wxs-reflection-003.md

```diff
@@ -0,0 +1,61 @@
++++
+title = 'Cultivating the Natural State'
+date = 2024-06-13T12:00:00-04:00
+authors = ["Xavier Snelgrove"]
++++
+
+I skipped last week's reflection, as I was on a retreat. Before you dock me a
+few points hear me out, there's a sense in which this retreat is field research
+for my project. The leader Bonnitta Roy is an independent scholar and
+philosopher who runs an online Zoom/Substack philosophy community called [The
+Pop-Up School](https://bonnittaroy.substack.com/). This retreat, on "Cultivating
+the Natural State" was an in-person gathering in rural Vermont of primarily
+people in the the Pop-Up School's community.
+
+Roys community is interesting on many levels. She's doing work which is academic
+in flavour, but because she does not have the constraints that the conventional
+academic discipline might impose, so there is also the feeling that you're
+participating in something *new*. Something that doesn't fit into the existing
+boxes, but perhaps rhymes with them.
+
+A theme of the retreat was on "left-brain right-brain integration", and this was
+reflected in its content which would run from embodiment exercises and group
+music jamming, all the way to very verbal and diagrammatic expositions of Roy's
+ongoing metaphysics development project. Her interests span this range, and
+therefore so does the community she attracts. The attendees were striking in
```

Comment:
>  Her interests span this range, and therefore so does the community she attracts.

I find this every salient!

We attract the community that reflects ourselves back onto ourselves - and in some sense cultivating yourself is a part of community building. The research you're doing is changing yourself which with reverberate to the other current fellows and to the future fellwoship, and as we build the community it will all feed into itself and grow...

Reply:
yes!